Modernize fix deprecated header

r-barnes opened the pull request:
Differential Revision: D29716965

malfet reviewed (approved):
Top commit on the stack (7e1a2aa648993e86da458e56f8333bfdcaf929d3) looks good to me

This PR is stacked on top of https://github.com/pytorch/pytorch/pull/61737

r-barnes committed:
Modernize some loops in torch

Differential Revision: D29716813

fbshipit-source-id: 6e0c11d0a4f57d1a7e2dc5a85f91133fc6c73372

r-barnes committed:
Modernize fix deprecated header (#61736)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/61736

Test Plan: Sandcastle

Reviewed By: malfet

Differential Revision: D29716965

fbshipit-source-id: 68c95171479a71348d892f0c38d2e2627e93af82
